db.go

```diff
@@ -13,7 +13,7 @@ type Migratable interface {
 type Postgres struct{}
 
 func (p Postgres) SelectMigrationTableSql() string {
-	return "SELECT tablename FROM pg_catalog.pg_tables WHERE tablename = $1"
+	return "SELECT table_name FROM information_schema.tables WHERE table_name= $1 AND table_catalog = $2"
```

Comment:
My testing showed that table_schema in PG isn't treated the same as in MySQL.  Below you will see that the table_schema doesn't represent the database to which we're connected, and that table_catalog does.

In fact PG may in fact actually restrict the table check query to only the given schema we're connected to.  (Which makes sense given that the documentation at the provided link states _"table_catalog - Name of the database that contains the table (always the current database)"_.)

> prompt:~/$ psql bar
> psql (9.1.15)

> bar=# SELECT table_name, table_catalog,table_schema FROM information_schema.tables where table_name='gomigrate';
> table_name | table_catalog | table_schema 
> ------------+---------------+--------------
> (0 rows)

> bar=# CREATE TABLE gomigrate (
> bar(#                   id           SERIAL       PRIMARY KEY,
> bar(#                   migration_id BIGINT       UNIQUE NOT NULL
> bar(#                 );

> CREATE TABLE
> bar=# SELECT table_name, table_catalog,table_schema FROM information_schema.tables where table_name='gomigrate';
> table_name | table_catalog | table_schema 
> ------------+---------------+--------------
> gomigrate  | bar           | public
> (1 row)


> prompt:~/$ psql foo
> psql (9.1.15)

> foo=# CREATE TABLE gomigrate (
> foo(#                   id           SERIAL       PRIMARY KEY,
> foo(#                   migration_id BIGINT       UNIQUE NOT NULL
> foo(#                 );

> CREATE TABLE
> foo=# SELECT table_name, table_catalog,table_schema FROM information_schema.tables where table_name='gomigrate';
>  table_name | table_catalog | table_schema 
> ------------+---------------+--------------
>  gomigrate  | foo           | public